backend/pkg/helm/release_test.go

```diff
@@ -35,10 +38,33 @@ func GetClient(t *testing.T, clusterName string) clientcmd.ClientConfig {
 	return clientcmd.NewInteractiveClientConfig(*config, clusterName, nil, nil, nil)
 }
 
+func getStatus(ch cache.Cache, action string, releaseName string, t *testing.T) (string, error) {
```

Comment:
We should match with other functions. 

```suggestion
func getStatus(t *testing.T, ch cache.Cache, action, releaseName string) (string, error) {
```